web/src/components/organisms/Settings/Members/hooks.ts

```diff
@@ -151,13 +151,20 @@ export default () => {
             }[role],
           },
         });
-        const workspace = results.data?.updateUserOfWorkspace?.workspace;
+        const workspace = result.data?.updateUserOfWorkspace?.workspace;
+        if (result.errors || !workspace) {
+          Notification.error({ message: t("Failed to update member's role.") });
+          return;
+        }
         if (workspace) {
           setWorkspace(fromGraphQLWorkspace(workspace as GQLWorkspace));
         }
+        if (result.data) {
+          Notification.success({ message: t("Successfully udpated member's role!") });
+        }
```

Comment:
can you rethink the if statements logic? something like this?

```suggestion
        const workspace = result.data?.updateUserOfWorkspace?.workspace;
        if (result.errors || !workspace) {
          Notification.error({ message: t("Failed to update member's role.") });
          return;
        }
        setWorkspace(fromGraphQLWorkspace(workspace as GQLWorkspace));
        Notification.success({ message: t("Successfully udpated member's role!") });
```

Reply:
Okay, I should have referred to `handleMemberRemoveFromWorkspace`, not `handleUsersAddToWorkspace`.
Thank you.

Reply:
Thank you!